src/modules/Involvementapi.js

```diff
@@ -7,6 +7,25 @@ function getAllLikes() {
     .then((data) => data);
 }
 
+const getArtworkComments = async (imageId) => {
+  const url = `${involvementAPI}apps/${involvementAPIid}/comments?item_id=${imageId}`;
```

Comment:
![Screenshot (227)](https://github.com/tchasinga/API-capstone-project/assets/121050688/56bdcf96-e0a8-4462-a5f4-e24a51f09283)

Reply:
kindly fix this issue it occur when I'm clicking on the comment bottom 
I suggested you to take reference point of your errors below 👎 

Error Handling: In the getArtworkComments function, the catch block is throwing a new error without providing the actual error message. It would be helpful to include the specific error message in the thrown error to aid in debugging. For example:

```
catch (error) {
  throw new Error(`Error fetching data: ${error.message}`);
}
```

Error Handling for HTTP Status Codes: In the getArtworkComments function, you're handling only the 404 and 400 status codes by returning an empty array. It would be beneficial to have a more comprehensive error handling mechanism that covers other possible status codes as well. You can consider adding a generic error message for all other non-success status codes. For example:

```
if (!response.ok) {
  if (response.status === 404 || response.status === 400) {
    return [];
  }
  throw new Error(`An error occurred: HTTP status ${response.status}`);
}
```

Reply:
@tchasinga Can you please add a console.log() and see what the error is? I was not able to reproduce this problem on my machine.

Reply:
The reason I handle only 404 and 400 error codes is because that is what the Involvement API returns if we ask for the comments of an artwork that has no comments yet. So in that case, it's really not an error in the sense a 500 error is.